geany.appdata.xml.in

```diff
@@ -0,0 +1,34 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<component>
+  <id type="desktop">geany.desktop</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <pkgname>geany</pkgname>
```

Comment:
Does not belong into the appdata tile - this is inserted by whatever procuces the distro appstream. Upstream has no influence / information about used downstream package names